.github/CODEOWNERS

```
@@ -40,6 +40,9 @@
 # verification team
 /roles/jenkins_job_launcher          @redhatci/verification
 /roles/junit2json                    @redhatci/verification
+/plugins/filter                      @redhatci/verification @redhatci/dci
+/tests/unit/filter                   @redhatci/verification @redhatci/dci
+/tests/unit/data                     @redhatci/verification @redhatci/dci
```

likewise, no need to have dci to review verification plugin tests.
```suggestion
/tests/unit/filter/*junit2obj*       @redhatci/verification
/tests/unit/data/*junit2obj*         @redhatci/verification
```